src/jetstream/plugins/userinfo/noauth_user.go

```diff
@@ -0,0 +1,57 @@
+package userinfo
+
+import (
+	"encoding/json"
+	"errors"
+	"net/http"
+
+	"github.com/cloudfoundry-incubator/stratos/src/jetstream/repository/interfaces"
+)
+
+// NoAuthUserInfo is a plugin to fetch user info
+type NoAuthUserInfo struct {
+	portalProxy interfaces.PortalProxy
+}
+
+// InitNoAuthUserInfo creates a new no auth user info provider
+func InitNoAuthUserInfo(portalProxy interfaces.PortalProxy) Provider {
+	return &NoAuthUserInfo{portalProxy: portalProxy}
+}
+
+// GetUserInfo gets info for the specified user
+func (userInfo *NoAuthUserInfo) GetUserInfo(id string) (int, []byte, *http.Header, error) {
+
+	uaaUser := &uaaUser{
+		ID:       id,
+		Origin:   "noauth",
+		Username: "admin",
```

Comment:
I don't think it makes much difference, but the `admin` string should have a single reference used here and in `authnone.go`